src/frontend/src/lib/components/networks/NetworksSwitcher.svelte

```diff
@@ -47,9 +46,8 @@
 			{/each}
 		</ul>
 
-		<div class="mb-4 mt-8 flex items-center justify-between">
-			<span class="px-4.5 font-bold">{$i18n.networks.show_testnets}</span>
-			<NetworksTestnetsToggle />
+		<div class="mb-5 mt-8 flex items-center justify-between">
```

Comment:
Not sure what is this change? The list of testenet should still be present in the network switcher when displayed.

Reply:
yes yes, it is, this specific part just removes the toggle from the list, when the switcher is displayed. Is that the question?

Reply:
Ahhhhh just the toggle. Gotcha.

- Do we still need the component `NetworksTestnetsToggle`?

- Do we sill need the above div `<div class="mb-5 mt-8 flex...` given that only a text will now be displayed?

Reply:
- Yes, we use it in Settings page the same one

- You are right, no, merged

Reply:
Great! Thanks